rezz.lic

```diff
@@ -1,118 +1,131 @@
-#   Documentation: https://elanthipedia.play.net/Lich_script_repository#rezz
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#rezz
+  
+  Define a waggle sets for rezz and rejuv if you don't want to use the default values in base-spells.yaml
+  
+  EXAMPLE
+  waggle_sets:
+    rezz:
+      Resurrection:
+        mana: 17
+    rejuv:
+      Rejuvenation:
+        mana: 10
+        cambrinth:
+        - 15
+
+  In order to infuse directly from your mana pool add the following to your yaml:
+  osrel_no_harness: true
+  
+=end
 
 custom_require.call(%w[common common-arcana events drinfomon spellmonitor])
 
-# rezz waggle set with POM if you want
-
 class Rezz
   include DRC
   include DRCA
-
+  
   def initialize
+    DRStats.cleric? ? nil : begin echo '*** Nice try, heathen! ***'; exit end
+	
     arg_definitions = [
       [
         { name: 'person', regex: /\w+/, description: 'Person to rezz' }
       ]
     ]
-
-    @rejuv_mana = 0
+	
     args = parse_args(arg_definitions)
     @settings = get_settings
-    rezz_prep(args.person)
+    
+    @person = args.person.capitalize
+	/the body of (#{@person}.*) who/ =~ bput('look people', /the body of (#{@person}.*) who/)
```

Comment:
It's still too greedy, if there is another person lying down after the target, it will pick up way too much.
New solution:
/the body of (#{@person}[a-z]*) who is lying down/

Reply:
What if this is an invasion triage and i want to raise a specific person in the room?

I agree with @jayRyan24, its already an arg to the script. What you could do is add an additional argument, or have the lack of a `person` argument tell the script to just raise anyone whose dead in the room.

Reply:
That's another option yeah, just `;rezz` to call it.

Reply:
Disregard, I misunderstood :)

Reply:
Oh, this is in case the body gets drug away, lol. Ok, I get it. I was like, wtf is the purpose of this and then I put his original comment with this, 2+2=4. 🥇 for my ability to understand.

Reply:
Just me being picky about having the person's full name for the fun of it.  I didn't know how to do string scan, and I'm sure I could have pulled the Also here line out of some lich generated data source somewhere.  I'm just not sure what variable stores that also here: line.

I didn't set it up to raise everyone, but I could do that.  I'd prefer to leave it as an optional argument.

Either ;rezz <target>
Or ;rezz all (or something like that.